src/main/java/CancelApplication/CancelApplicationInputBoundary.java

```diff
@@ -0,0 +1,5 @@
+package CancelApplication;
+
+public interface CancelApplicationInputBoundary {
+    CancelApplicationResponseModel cancelApplication(CancelApplicationRequestModel requestModel);
```

Comment:
I'm glad you caught that we should do a more specific method name rather than "create". cancelApplication is a good name as it is specific to your use case.